server/resolvers/UpdateEvent/createUpdateEventResolver.js

```diff
@@ -0,0 +1,16 @@
+const {
+  createUpdateEvent
+} = require('../../models/services/UpdateEventService');
+
+const createUpdateEventResolver = (
+  _,
+  { description, type = 'GENERAL' },
+  { transaction }
+) => {
```

Comment:
Leaving feedback for myself, add `auth` for this and `updateEventResolver` or remove graphql resolver altogether since it is not used currently

Reply:
[764e366](https://github.com/w3c/aria-at-app/pull/1348/commits/764e36692b24fed3da1fabe089afd12ca2884b97)